tests/components/flick_electric/test_init.py

```diff
@@ -21,7 +21,7 @@ async def test_init_auth_failure_triggers_auth(hass: HomeAssistant) -> None:
             "homeassistant.components.flick_electric.HassFlickAuth.async_get_access_token",
             side_effect=AuthException,
         ),
-        patch.object(hass.config_entries.flow, "async_init") as mock_flow_init,
+        # patch.object(hass.config_entries.flow, "async_init") as mock_flow_init,
```

Comment:
Let's not have commented out code

Reply:
Whoops, that was me testing it was working 😅